code-challenges.test.js

```diff
@@ -22,6 +22,28 @@ const secretCodeWord2 = "Gobbledygook"
 const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("secretCodedWord1", () => {
+
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("returns a coded string", () => {
+
+        //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+        expect(secretCodeWord1()).toEqual("L4ck4d41s1c4l")
+    })
+})
+
+describe("secretCodedWord2", () => {
+    it("returns a coded string", () => {
+        expect(secretCodeWord2()).toEqual("G0bbl3dyg00k")
+    })
+})
+
+describe("secretCodedWord3", () => {
+    it("returns a coded string", () => {
+        expect(secretCodeWord3()).toEqual("3cc3ntr1c")
+    })
+})
```

Comment:
I understand your approach here, but you can make this section a bit shorter by creating a single test and passing in all three expect statements. 
```
describe("secretCodedWord3", () => {
     const secretCodeWord1 = "Lackadaisical"
     const secretCodeWord2 = "Gobbledygook"
     const secretCodeWord3 = "Eccentric"
     it("returns a coded string", () => {
        expect(secretCodeWord1()).toEqual("L4ck4d41s1c4l")
        expect(secretCodeWord2()).toEqual("G0bbl3dyg00k")
        expect(secretCodeWord3()).toEqual("3cc3ntr1c")
    })
})
```